apps/api-journeys/prisma/factories/nua9.ts

```diff
@@ -0,0 +1,1527 @@
+      extraAttrs: {
+        label: ''
+      },
+      parentOrder: 2
+    }
+  })
+
+  //   Yes, I would step/option
+  //   First part of the prayer
+  const stepPrayer1 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 3
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'RadioOptionBlock',
+      parentBlockId: question1.id,
+      extraAttrs: {
+        label: 'Yes I would',
+        action: {
+          gtmEventName: 'click',
+          blockId: stepPrayer1.id
+        }
+      },
+      parentOrder: 0
+    }
+  })
+  const prayerImageId1 = uuidv4()
+  const prayerCard1 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'CardBlock',
+      parentBlockId: stepPrayer1.id,
+      extraAttrs: {
+        themeMode: ThemeMode.dark,
+        themeName: ThemeName.base,
+        coverBlockId: prayerImageId1
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      id: prayerImageId1,
+      journeyId: journey.id,
+      blockType: 'ImageBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        src: 'https://images.unsplash.com/photo-1433324168539-154874446945?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1170&q=80',
+        alt: 'Decision',
+        width: 1920,
+        height: 1080,
+        blurhash: 'L5AVCm=Z4VIW004T.Awb8w_2b_jE'
+      },
+      parentOrder: 0
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        content: '1/3',
+        variant: 'h6',
+        color: 'primary',
+        align: 'right'
+      },
+      parentOrder: 1
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'TypographyBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        content:
+          'Jesus, thank you for loving me, thank you for the worth you speak into my life by dying for my sin.',
+        variant: 'subtitle1',
+        color: 'primary',
+        align: 'left'
+      },
+      parentOrder: 2
+    }
+  })
+
+  // Second part of the prayer
+  const stepPrayer2 = await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'StepBlock',
+      extraAttrs: {
+        locked: false,
+        nextBlockId
+      },
+      parentOrder: 4
+    }
+  })
+  await prisma.block.create({
+    data: {
+      journeyId: journey.id,
+      blockType: 'ButtonBlock',
+      parentBlockId: prayerCard1.id,
+      extraAttrs: {
+        label: 'Next',
+        variant: 'contained',
+        color: 'primary',
+        size: 'small',
+        startIcon: {
+          name: 'PlayArrow'
```

Comment:
```suggestion
          name: 'ChevronRight'
```

If it's available - else add to our IconList during cooldown